python/doc/examples/data_analysis/draw_survival.ipynb

```diff
@@ -0,0 +1,644 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Draw the survival of a sample or a distribution\n",
+    "\n",
+    "## Introduction\n",
+    "\n",
+    "The goal of this example is to show how to draw the survival function of a sample or a distribution, in linear and logarithmic scales. \n",
+    "\n",
+    "Let $X$ be a random variable with distribution function $F$:\n",
+    "$$\n",
+    "F(x) = P(X\\leq x)\n",
+    "$$\n",
+    "for any $x\\in\\mathbb{R}$. \n",
+    "The survival function $S$ is:\n",
+    "$$\n",
+    "S(x) = P(X>x) = 1 - P(X\\leq x) = 1 - F(x)\n",
+    "$$\n",
+    "for any $x\\in\\mathbb{R}$. \n",
+    "\n",
+    "Let's assume that $\\{x_1,...,x_N\\}$ is a sample from $F$. \n",
+    "\n",
+    "Let $\\hat{F}_N$ be the empirical distribution function:\n",
+    "$$\n",
+    "\\hat{F}_N(x) = \\frac{1}{N} \\sum_{i=1}^N \\mathbb{1}_{x_i\\leq x}\n",
```

Comment:
\mathbb{1} is not well-defined (see [here](https://tex.stackexchange.com/questions/488/blackboard-bold-characters/3260). Either you manage to implement one of the solutions given in the link in the context of a notebook, or you resort to another way to display the indicator function, e.g `\\mathbf{1}`

Reply:
I did'nt know; this is ugly. Let's go for mathbf.